.circleci/config.yml

```diff
@@ -9,16 +9,15 @@ orbs:
 jobs:
   "python-3_7":
     docker:
-      - image: circleci/python:3.7
+      - image: cimg/python:3.7
 
     working_directory: ~/repo
 
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "test-requirements.txt" }}
-            - v1-dependencies-
```

Comment:
It appears that a broader cache was obstructing builds caused by changes to the Docker image. Is this needed? 🤔